src/components/VTimePicker/VTimePickerClock.js

```diff
@@ -102,21 +92,25 @@ export default {
         this.update(value)
       }
     },
+    isInner (value) {
+      return this.double && (value - this.min >= this.roundCount)
+    },
     handScale (value) {
-      return this.double && (value - this.min >= this.roundCount) ? (this.innerRadius / this.radius) : (this.outerRadius / this.radius)
+      return this.isInner(value) ? this.innerRadius : 1
     },
     isAllowed (value) {
       return !this.allowedValues || this.allowedValues(value)
     },
-    genValues () {
+    genValues (h) {
```

Comment:
this is inconsistent with other parts of the framework, use `this.$createElement`